etc/cas.properties

```diff
@@ -105,12 +179,35 @@ host.name=cas01.example.org
 #
 # It is often time helpful to externalize log4j.xml to a system path to preserve settings between upgrades.
 # e.g. log4j.config.location=/etc/cas/log4j.xml
-log4j.config.location=/etc/cas/log4j.xml
+log4j.config.location=file:///etc/cas/log4j.xml
```

Comment:
4.1 uses log4j v2. This isnt the right config. Please follow up with a separate PR.